docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs

```diff
@@ -86,9 +93,12 @@ public class TensorData
         public static TensorData[] GetTensorData()
         {
             // This can be any numerical data. Assume image pixel values.
-            var image1 = Enumerable.Range(0, inputSize).Select(x => (float)x / inputSize).ToArray();
-            var image2 = Enumerable.Range(0, inputSize).Select(x => (float)(x + 10000) / inputSize).ToArray();
-            return new TensorData[] { new TensorData() { input = image1 }, new TensorData() { input = image2 } };
+            var image1 = Enumerable.Range(0, inputSize).Select(
+                x => (float)x / inputSize).ToArray();
+            var image2 = Enumerable.Range(0, inputSize).Select(
+                x => (float)(x + 10000) / inputSize).ToArray();
```

Comment:
As discussed new line between any two lines that were broken into multiple lines.